.gitignore

```diff
@@ -22,4 +22,5 @@ docs/venv
 
 # development
 .vscode
+.venv
```

Comment:
This might be specific to your personal setup. I think better to avoid doing this.